libs/platform/experience/src/plugins/theme/theme-tokens.ts

```diff
@@ -201,10 +202,32 @@ export class ThemeTokens {
       let tokenMedia = DesignTokenGlobal.Global as string;
 
       if (!tokens.media && tokens.color) {
-        tokensList.push({
-          media: { [DefaultMedia.Mode]: 'light' },
-          color: tokens.color,
-        });
+        for (const [key, color] of Object.entries(tokens.color)) {
+          // TODO: delete when all colors will be replaced
```

Comment:
still needed?

Reply:
Yes, until we have these fallbacks
<img width="401" alt="image" src="https://user-images.githubusercontent.com/16482929/235731651-1fad2980-bff6-4223-9850-35697c2d3d2d.png">